new_event.md

```diff
@@ -0,0 +1,248 @@
+
+Select your namespace from the dropdown on the left and check that the deployment is OK and two pods have successfully been created.
+
+## Setup load balancer
+
+Browse to AWS Management Console > EC2 > Load Balancers and find the new load balancer.
+
+It won't have a useful name so click on a load balancer and select the `Tags` tab which will show you the service name.
+
+Once you have found it you need the DNS name from the `Description` tab
+
+Log in to Cloudflare using the credentials for the `cpwebteam@coderdojo.com` account in lastpass.
+
+Go to `coolestprojects.org` and then `DNS`
+
+Add a record to create the CNAME for the new site:
+
+- Type: CNAME
+- Name: The subdomain for the new event e.g. `register-uk-2020`
+- IPv4 Address: The DNS name of the load balancer
+- Proxy Status: DNS Only (Leave TTL as Auto)
+
+---
+
+## Create Event in Database
+
+To create the event you will need all the information around dates and times, venue, categories etc.
+
+### Prepare Database
+
+Open the kubernetes psql shell:
+
+`kubectl exec -it psql-shell /bin/bash`
+
+Connect to your newly created database using the details in the db.json to create a connection string:
+
+`psql 'postgresql://username:password@hostname/database-name'`
+
+Add the `uuid-ossp` extension to database to allow UUIDs to be generated:
+
+`CREATE EXTENSION IF NOT EXISTS "uuid-ossp";`
+
+### Create Event
+
+While connected to the psql shell create an empty event using the following script:
+
+```
+INSERT INTO event (id) VALUES (Uuid_generate_v4());
+```
+
+Get the ID of the newly created event `select id from event;` (there should only be one event in the database)
+
+Then update the event information to the correct values:
+
+(See notes below about formatting the data)
+
+```
+UPDATE event
+SET    name = '',
+       slug = '',
+       location = '',
+       date = '',
+       registration_start = '',
+       registratin_end = '',
+       categories = '',
+       homepage = '',
+       contact = '',
+       questions = '',
+       freeze_date = '',
+       tz = '',
+WHERE  id = '**id of new event**';
+```
+
+##### Formatting Data Input
+
+It's probably easiest to input dates in the ISO format and UTC `'2020-12-25 09:30:00Z`
+
+Remember to allow for daylight savings as necessary.
+
+**Categories** is stored as json with a category code as the key, and the name as the value e.g.
+
+```
+'{"HW": "Hardware", "GAM": "Games"}'
+```
+
+**Questions** is an array of string keys the front end uses to show the correct question to the users. The keys and their related questions are in `client/src/project/CustomQuestions.js` e.g.
+
+```
+'["travel_bursary", "attendance_prev_year"]'
+```
+
+**tx** is the timezone, the frontend displays this so it needs to be correct. For UK it will be `'Europe/London'`, for the international event in Dublin it will be `'Europe/Dublin'`. The timezone for the US event will need to be provided or found.
```

Comment:
```suggestion
**tz** is the timezone, the frontend displays this so it needs to be correct. For UK it will be `'Europe/London'`, for the international event in Dublin it will be `'Europe/Dublin'`. The timezone for the US event will need to be provided or found.
```